json-ld/README.md

```diff
@@ -0,0 +1,3 @@
+# json-ld
+
+This is JSON-LD library.
```

Comment:
The description could be a bit more verbose ;)

Reply:
Good point. Forgot about it :) Added some now.